package.json

```diff
@@ -111,7 +111,7 @@
   "lint-staged": {
     "*.js": [
       "eslint --cache --fix",
-      "prettier --write"
+      "prettier --ignore-path ./.gitignore --write ."
```

Comment:
prettier wasn't even working because it was missing `.` after `--write`.
Also ignoring folders/files mentioned in .gitignore from prettier

Reply:
Removed it